docs/bbe-guidelines/bbe-guidelines.md

```diff
@@ -0,0 +1,174 @@
+
+
+![alt_text](images/Guidelines-in4.png "image_tooltip")
+
+
+b.io:
+
+
+
+<p id="gdcalert6" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/Guidelines-in5.png). Store image on your image server and adjust path/filename if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert7">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/Guidelines-in5.png "image_tooltip")
+
+
+
+
+*   **We use a common pattern in `.out` files. Use the following format and customize only when necessary (ex: When needed to add more command line args etc)**
+
+<span style="text-decoration:underline;">For an example with main:</span>
+
+# To run this sample, navigate to the directory that contains the
+
+# `.bal` file and issue the `ballerina run` command.
+
+$ ballerina run <sample_file_name>.bal
+
+<span style="text-decoration:underline;">For an example with a service:</span>
+
+# To start the service, navigate to the directory that contains the
+
+# `.bal` file and issue the `ballerina run` command.
+
+$ ballerina run hello_world_service.bal
+
+
+
+*   **As a practice, we use `<code>ballerina/io`</code> methods in main examples and `<code>ballerina/log</code>` in services examples. Don’t use both `<code>io:println</code>` and `<code>log:printInfo</code>` in the same sample.</strong>
+*   <strong>Remove unused imports in `.bal` files.</strong>
+*   <strong>Format the ballerina source using an IDE Plugin. </strong>
+*   <strong>Always think about the length of the sample. A good example should be short.</strong>
+*   <strong>If a new example is added/deleted, update the [index.json](https://github.com/ballerina-platform/ballerina-lang/blob/master/examples/index.json) file as well.</strong>
+
+
+
+**<span style="text-decoration:underline;">Folder Structure</span>**
+
+
+
+*   **Each BBE is in a separate directory with the sample name. Folder name should be all lowercase and words are separated by “-”. (ex: abstract-objects)**
+*   **Each example should contains at least following files. File name should be same as folder name, but “-” is replaced with underscore. (ex: abstract_objects)**
+
+
+
+<p id="gdcalert7" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/Guidelines-in6.png). Store image on your image server and adjust path/filename if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert8">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/Guidelines-in6.png "image_tooltip")
+
+
+           .bal - Sample code to display in b.io.
```

Comment:
Same comment as above, regarding the use of abbreviations

Reply:
Changed to "[Ballerina website](https://ballerina.io/)".